.github/workflows/release.yml

```diff
@@ -1,90 +1,70 @@
+name: Publish plerkle plugin 
+
 on:
+  workflow_dispatch:
   push:
     tags:
-      - "v*"
+      - 'v*'
 
 env:
   CARGO_TERM_COLOR: always
-  IMAGE_NAME: plerkle-test-validator
-  RUST_VERSION: 1.69.0
-  SOLANA_VERSION_STABLE: v1.16.6
+
 jobs:
-  release-stable:
-    runs-on: buildjet-8vcpu-ubuntu-2004
+  build20:
```

Comment:
Since the `build20` job now has a "matrix" strategy, would this run twice (one for each OS) and therefore create 2 releases? Not sure if this is intentional.

Reply:
It can be do on both ways, that's a thing you may decide, I can publish to one release, and rename things (that's the current workflow). Or I can make separated workflows and releases.